types.d.ts

```diff
@@ -0,0 +1,7 @@
+export enum ReactModuleContainerError {
```

Comment:
Another advantage of using custom errors is that this file wouldn't be needed.

Reply:
It would still be needed if we want to use the error `name`s